tests/Controller.spec.ts

```diff
@@ -780,8 +780,8 @@ describe('Controller spec', () => {
             )
           ).value.ok.ok,
           {
-            collateral: 0,
-            shortfall: 500 - 450,
+            collateral: 450,
+            shortfall: 0,
```

Comment:
Same
> This fix should be a code optimization, not a feature change, but do you see what the difference is here?

Furthermore, the state of the capacity is reversed here, which is of particular concern.